pkg/ha/filter.go

```diff
@@ -60,7 +59,6 @@ func (h *Filter) Process(_ *http.Request, wr *prompb.WriteRequest) error {
 	if !minT.Before(leaseStart) {
 		if !allowInsert {
 			wr.Timeseries = wr.Timeseries[:0]
-			log.Debug("msg", "the samples aren't from the leader prom instance. skipping the insert", "replica", replicaName)
```

Comment:
Why are we dropping this and the one below this?

Reply:
To drop the noise from HA debug logs... When running Promscale in debug mode I see too many debug logs from HA. So we decided to drop them.